### DISCUSSION TITLE
firefly-iii install script

### DISCUSSION THREAD

**Discussion by ghost** [firefly-iii/firefly-iii#4475] [General]
Hey everyone,

I've wrapped the install up into a nifty script.  @JC5 , you're welcome to add it to your repo, and let me know if you find any issues :)

https://github.com/edwardsj9090/firefly-iii-automation

**Comment by JC5** [comment]
Very cool! :tada:

I'll take some time and link to it from the installation guide!

**Reply by ghost** [reply]
Awesome.  Hope it helps :)

**Reply by ghost** [reply]
Made some slight changes.
- Added a line in the script to enable the apache rewrite module.

**Reply by ghost** [reply]
Added a copy of the apache2.conf file with the necessary changes to protect the firefly-iii folder and allow access to /public/.  The script copies the file to the /etc/apache2/ directory.

**Reply by JC5** [reply]
https://github.com/firefly-iii/docs/blob/main/docs/docs/firefly-iii/installation/self_hosted.md

Will be a while for the docs to update, but it's there!

**Reply by ghost** [reply]
Oh nice!  Thank you so much man.  Well I hope it helps.  I went through a few trials before getting it right, but it's working very well now :)
